File: frontend/lib/models/club_create_dto.dart
class ClubCreateDto {
  final String name;
  final String address;
  final int? lanesCount;
  final String? contactPhone;
  final String? contactEmail;

  ClubCreateDto({
    required this.name,
    required this.address,
    this.lanesCount,
    this.contactPhone,
    this.contactEmail,
  });

  Map<String, dynamic> toJson() {
    return {
      'name': name,
      'address': address,
      'lanesCount': lanesCount,
      'contactPhone': contactPhone,
      'contactEmail': contactEmail,
    };
  }
}

File: backend/CoreService/src/main/java/ru/bowling/bowlingapp/DTO/ClubCreateDTO.java
package ru.bowling.bowlingapp.DTO;

import jakarta.validation.constraints.NotBlank;
import lombok.Data;

@Data
public class ClubCreateDTO {

    @NotBlank
    private String name;

    @NotBlank
    private String address;

    private Integer lanesCount;

    private String contactPhone;

    private String contactEmail;
}
